test/integration/tests/lusdPSM.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  AavePCVDeposit,
+  MintRedeemPausePSM,
+  Fei,
+  IERC20,
+  PCVDripController,
+  PSMRouter,
+  WETH9,
+  BAMMDeposit
+} from '@custom-types/contracts';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { BigNumber } from 'ethers';
+import hre, { ethers } from 'hardhat';
+import { NamedAddresses, NamedContracts } from '@custom-types/types';
+import {
+  expectRevert,
+  getImpersonatedSigner,
+  increaseTime,
+  overwriteChainlinkAggregator,
+  resetFork
+} from '@test/helpers';
+import proposals from '@test/integration/proposals_config';
+import { TestEndtoEndCoordinator } from '../setup';
+import { forceEth } from '../setup/utils';
+
+const oneEth = ethers.constants.WeiPerEther;
+const toBN = ethers.BigNumber.from;
+
+before(async () => {
+  chai.use(CBN(ethers.BigNumber));
+  chai.use(solidity);
+  await resetFork();
+});
+
+describe('lusd PSM', function () {
```

Comment:
I don't think this should be an entirely new integration test, can we move it to the peg one or something?

Reply:
every other psm had it's own IT, this shouldn't be any different

Reply:
I think its a bad pattern and am making this suggestion to change, before entrenching a suboptimal solution. Lets discuss from first principles not anchor to precedent. What do you think?

Reply:
what file are you proposing we move this into? psm.ts?